docs/website/docs/plus/getting-started/installation.md

```diff
@@ -74,46 +75,46 @@ This way, all the dependencies for your current project will be isolated from pa
 
   <TabItem value="ubuntu">
 
-Create a new virtual environment in your working folder. This will create an `./env` directory where your virtual environment will be stored:
+Create a new virtual environment in your working folder. This will create an `./venv` directory where your virtual environment will be stored:
```

Comment:
```suggestion
Create a new virtual environment in your working folder. This will create a `./venv` directory where your virtual environment will be stored:
```